### PR TITLE
New uninhabited planet: Donya

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Content (Planet)**

## Summary
Added a new planet in Sabik called Donya.
Delta re-PR of https://github.com/endless-sky/endless-sky/pull/10529. Probably should have included this in #104.

This would be one of the colonies that the "billionaire philanthropists" tried to start. However, here they were not as lucky as on planets like Longjump.

## Screenshots
donya

The landscape image and description of the planet.

## Testing Done
Took screenshot, landed, launched, landed.

## Save File
This save file can be used to test these changes:
[Asiah Racer~Donya.txt](https://github.com/user-attachments/files/17199144/Asiah.Racer.Donya.txt)

Load the save, take off, jump, and land. The travel course is already mapped out.

### CODE REVIEW

File: data/map planets.txt [hunk 1]
```diff
@@ -1139,6 +1139,13 @@ planet "Dlia Jzaur"
 	description `	Oddly enough, Dlia Jzaur has many unnatural looking formations on the surface that seem to reach up to the sky in areas where eclipses often occur. The formations are hollow underneath, but show no signs of Gegno involvement.`
 	government Uninhabited
 
+planet "Donya"
+	attributes uninhabited
+	landscape land/fields29
+	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic in the eyes of early settlers and the billionaire philanthropists who funded them.`
+	description `	Despite the beauty and the funding, it remains unsettled, as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists to scrub the skies of the rain seems to have adjusted to the terrible liquid quite well, although it hasn't managed to entirely eliminate it. The planters, who couldn't bear the dreadful weather, left when they discovered that they couldn't stop the rains.`
```
Comment (Zitchas) [inline data/map planets.txt]:
```suggestion
	description `	Despite the beauty and the funding, it remains unsettled, as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists to scrub the skies of the rain seems to have adjusted to the terrible liquid quite well, although it hasn't managed to eliminate it. The planters, who couldn't bear the dreadful weather, left when they discovered that they couldn't stop the rains.`
```

Just removing one word. This makes it less friendly. If the plants had even halved the amount of acid rain, someone would have stuck around out of stubborness or something.

File: data/map systems.txt [hunk 2]
```diff
@@ -30223,8 +30223,12 @@ system Sabik
 		period 513.620
 	object
 		sprite planet/gas4-b
-		distance 2825.19
-		period 1526.877
+		distance 1280.19
+		period 732.877
```
Comment (Zitchas) [inline data/map systems.txt:30227]: Why is this gas giant planet being changed?

**Comment by Zitchas** [comment]
Overall, I like it. I'm generally of the opinion that every planet should be landable. Every single one of them (provided the ship has the equipment to do so. Not every planet needs to be landable to every ship.)

Having a failed colony on something that looks beautiful is a nice touch.

**Comment by TheGiraffe3** [comment]
> Overall, I like it. I'm generally of the opinion that every planet should be landable. Every single one of them (provided the ship has the equipment to do so. Not every planet needs to be landable to every ship.)

Would be really fun - but the amount of work required to generate files for _every single planet_ would be massive.
Makes me wonder if it would be possible to do something like No Man's Sky procedural generation system.

**Comment by Zitchas** [comment]
Have you played Mass Effect, by any chance? Every planet in the game has a description. Most planets one can only orbit and maybe do a scan, but the amount of weird lore, mysteries, and whatnot is impressive.

Doing so for every planet in ES would, agreed, be a truly massive undertaking. But have several hundred people write two or three planet descriptions each... That is very doable. And most of them don't have to be anything far out there. Some could be simple and mundane, in fact, many should be. We could have easter egg oblique references to basically every Sci-Fi series in existence somewhere out there...

**Comment by TheGiraffe3** [comment]
> Have you played Mass Effect, by any chance? Every planet in the game has a description. Most planets one can only orbit and maybe do a scan, but the amount of weird lore, mysteries, and whatnot is impressive.

No, unfortunately.
I saw that you mentioned a while back how you wished that everybody on the Discord server would write a mission chain and submit it, that we might end up with 100 Puny duplicates but it would add so much flavor... too bad that's unlikely to happen with planet descriptions.
Mainly because of overusing landscape images.

**Comment by Zitchas** [comment]
I am sure there must be at least a couple hundred alien landscape images out there that are compatible with ES. It is just a matter of finding them.

**Pull request merged**